chart/values.schema.json

```diff
@@ -8952,6 +8952,325 @@
                 }
             }
         },
+        "databaseCleanup": {
+            "description": "This runs as a CronJob to cleanup the database after the retention period.",
+            "type": "object",
+            "x-docsSection": "Jobs",
+            "additionalProperties": false,
+            "properties": {
+                "enabled": {
+                    "description": "Enable database cleanup.",
+                    "type": "boolean",
+                    "default": false
+                },
+                "schedule": {
+                    "description": "Database cleanup schedule (templated).",
+                    "type": "string",
+                    "default": "0 0 * * 0"
+                },
+                "retentionDays": {
+                    "description": "Number of days to retain records in the metadata database.",
+                    "type": "integer",
+                    "default": 90
+                },
+                "command": {
+                    "description": "Command to use when running the database cleanup cronjob (templated).",
+                    "type": [
+                        "array",
+                        "null"
+                    ],
+                    "items": {
+                        "type": "string"
+                    },
+                    "default": null
```

Comment:
Would not `"bash"` go into here from args below?

Reply:
its not necessary, but possible if you want to be explicit